frontends/mit-open/src/page-components/SearchDisplay/ResourceCategoryTabs.tsx

```diff
@@ -8,16 +8,30 @@ import {
 } from "ol-components"
 import { ResourceCategoryEnum, LearningResourcesSearchResponse } from "api"
 
-const TabsList = styled(TabButtonList)({
+const TabsList = styled(TabButtonList)(({ theme }) => ({
   ".MuiTabScrollButton-root.Mui-disabled": {
     display: "none",
   },
-})
+  [theme.breakpoints.up("md")]: {
```

Comment:
We specify the tab width because it doesn't look good when the tab sizes change as the user selects and unselects facets. On mobile the tabs are mostly are covered by the facet selection drawer when facets are changed so that isn't an issue. Also space is more valuable on mobile as you said. "md" is the breakpoint when the site switches from displaying the facets on the right side vs a drawer that slides in and out.